services/server/server.go

```diff
@@ -321,78 +316,6 @@ func LoadPlugins(ctx context.Context, config *srvconfig.Config) ([]*plugin.Regis
 			return local.NewStore(ic.Root)
 		},
 	})
-	plugin.Register(&plugin.Registration{
```

Comment:
If possible, we can move the content plugin init out of services package.

Reply:
Yes, any plugin which isn't dynamically created via configuration should be moved out